### DISCUSSION TITLE
Pest and Symfony

### DISCUSSION THREAD

**Discussion by antiftw** [pestphp/pest#1024] [Q&A] [answered]
Hi,

Recently I saw a video about this framework, and noticed a feature that I really like - the architecture test - so I thought I'd try it out. However when implementing a test (an exact copy from a PHP Annotated [video](https://www.youtube.com/watch?v=Vjjht-DMX3s&t=42m55s)):

`test('globals')->expect(['dd', 'dump'])->not->toBeUsed();`

I got an error:

`Error : Call to a member function toBeUsed() on array`

Which made me wonder, is this only available in Laravel? Or should I be able to use it in Symfony too? 

And if not, will this functionality be available in Symfony in the future? 

Thanks

**Comment by nunomaduro** [comment, marked as answer]
What version of Pest are you using?

**Reply by antiftw** [reply]
Apparently not the latest :) Thanks for the quick response, and apologies for the confusion.
